orca-core/src/test/groovy/com/netflix/spinnaker/orca/batch/lifecycle/LinearStageSpec.groovy

```diff
@@ -46,9 +45,9 @@ class LinearStageSpec extends AbstractBatchLifecycleSpec {
   @Shared
   def stageNavigator = new StageNavigator(Stub(ApplicationContext))
 
-  List<StepExecutionListener> listeners = [
-    new SpringBatchStageListener(executionRepository, new StageStatusPropagationListener())
-  ]
+//  List<StepExecutionListener> listeners = [
```

Comment:
Yeah, it comes from the SpringBatchExecutionListenerProvider now — see below. I'll remove the commented out code.